internal/osbuildexecutor/export_test.go

```diff
@@ -0,0 +1,8 @@
+package osbuildexecutor
+
+var ExtractOutputArchive = extractOutputArchive
```

Comment:
(comment, no action needed) this could also be written as:
```go
var (
   ExtractOutputArchive = extractOutputArchive
   FetchOutputArchive = fetchOutputArchive
   ...
)
```